redhataccess/pantheon
# CCS-3717: Updates to frontend build

This branch is to allow the React dev process to work without rebuilding any Karaf or Java resources. This enables the webpack server compile JUST the react/frontend components on when they're saved.

This also resolves CORS errors by running setting up a proxy with Spandx.

Testing instructions:
* Have Karaf built/working
* Follow "Getting Started" instructions from: `pantheon-bundle/frontend/README.md`
* Make a change in a file in React, confirm that dev server reloads browser and adds change
* Make sure I didn't mess up any existing way that you all are loading/using the React frontend
